### ISSUE TITLE
ToolTip should not be declared on an action

### ISSUE THREAD

**Issue by ratijas** [najepaliya/kleaner#1]
https://github.com/najepaliya/kleaner/blob/2f1c78a4f32eed9ee1c1d69fe691bd614005938d/src/contents/ui/HomePage.qml#L14-L16

Attached ToolTips work on graphical Items, not on abstract actions. Action doesn't have `pressed` or `hovered` properties either. What you actually see in a running app is a custom ToolTip instance in ActionToolBar's [PrivateActionToolButton.qml](https://invent.kde.org/frameworks/kirigami/-/blob/67fd90d1f08fd30de29a519d018ed6759d62fa7f/src/controls/private/PrivateActionToolButton.qml#L89-102)

Basically, these three bindings should be just removed.

**Comment by najepaliya** [comment]
Done.
